eng/package.ps1

```diff
@@ -12,6 +12,30 @@ $csproj = Join-Path $PSScriptRoot ../src/DotNet/DotNet.csproj
 
 if ($IsWindows)
 {
+    if (-not $msbuild)
+    {
+        # If MSBuild path isn't specified, try various locations
+        $vseditions = @('Enterprise', 'Preview', 'Internal', 'Community')
```

Comment:
Could this invoke `vswhere` instead of listing these?

https://docs.microsoft.com/en-us/visualstudio/install/tools-for-managing-visual-studio-instances

I think that is what the Cake equivalent does.

Reply:
I considered that, but that assumes `vswhere` is in the path, which it typically isn't (unless you use a Developer Command Prompt).

Reply:
Oh I see, vswhere is actually in a standard location, so I can always call it! I'll try that instead.